pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go

```diff
@@ -438,6 +438,14 @@ func (admitter *VMsAdmitter) validateVolumeRequests(vm *v1.VirtualMachine) ([]me
 		if len(causes) > 0 {
 			return causes, nil
 		}
+
+		if vmi.Status.MigrationState != nil && !vmi.Status.MigrationState.Completed {
+			return []metav1.StatusCause{{
+				Type:    metav1.CauseTypeFieldValueNotSupported,
+				Message: fmt.Sprintf("Cannot handle volume requests while VMI migration is in progress"),
+				Field:   k8sfield.NewPath("spec").String(),
+			}}, nil
+		}
```

Comment:
for this condition, can you use the `pkg/util/migrations/migrations.go` file's IsMigrating function to determine if a migration is in flight?

I'm not 100% sure Completed is always an accurate indicator.

Reply:
NP, done. Replaced the condition with `IsMigrating`.